docs/source/guides/data.rst

```diff
@@ -47,23 +47,21 @@ Read :ref:`this <datamodules>` for more details on LightningDataModule.
 Multiple Datasets
 *****************
 
-There are a few ways to pass multiple Datasets to Lightning:
+There are a few ways to pass multiple datasets to Lightning:
 
-1. Create a DataLoader that iterates over multiple Datasets under the hood.
-2. In the training loop you can pass multiple DataLoaders as a dict or list/tuple and Lightning
-   will automatically combine the batches from different DataLoaders. You can control the way how dataloaders of different length
-   are combined by the flag `multiple_trainloader_mode` of the :class:`~pytorch_lightning.trainer.Trainer`. Alternatively, you can provide
-   dataloaders via :class:`~pytorch_lightning.trainer.supporters.CombinedLoader`.
-3. In the validation, test or prediction you have the option to either return multiple DataLoaders as list/tuple, which Lightning will call sequentially,
-   or combine the dataloaders using :class:`~pytorch_lightning.trainer.supporters.CombinedLoader`, which Lightning will
+1. Create a DataLoader that iterates over multiple datasets under the hood.
+2. In the training loop, you can pass multiple DataLoaders as a dict or list/tuple, and Lightning will
+   automatically combine the batches from different DataLoaders.
+3. In the validation, test or prediction, you have the option to return multiple DataLoaders as list/tuple, which Lightning will call sequentially
+   or combine the DataLoaders  using :class:`~pytorch_lightning.trainer.supporters.CombinedLoader`, which Lightning will
    automatically combine the batches from different DataLoaders.
 
 
 Using LightningDataModule
 =========================
 
-You can set more than one :class:`~torch.utils.data.DataLoader` in your :class:`~pytorch_lightning.core.datamodule.LightningDataModule` using its dataloader hooks
-and Lightning will use the correct one under-the-hood.
+You can set more than one :class:`~torch.utils.data.DataLoader` in your :class:`~pytorch_lightning.core.datamodule.LightningDataModule` using its DataLoader hooks
```

Comment:
```suggestion
You can set more than one :class:`~torch.utils.data.DataLoader` in your :class:`~pytorch_lightning.core.datamodule.LightningDataModule` using its dataLoader hooks
```
it's `val_dataloader`, `test_dataloader`, so on....

Reply:
why capital L?

Reply:
updated.. I just updated `D` before :)